public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -0,0 +1,350 @@
+include ../_util-fns
+
+:marked
+  Angular's _internationalization_ ("_i18n_") tools help make your app availble in multiple languages.
+  
+  <a id="top"></a>
+  ## Table of contents
+    
+    * [Angular and i18n template translation](#angular-i18n)
+    * [Mark text with the _i18n_ attribute](#i18n-attribute)
+    * [Extract text with ng-xi18n](#ng-xi18n)
+    * [Translate](#translate)
+    * [Merge the translation file into the app](#merge)
+      * [JiT configuration](#jit)
+      * [AoT configuration](#aot)
+
+:marked
+  **Try this** <live-example>live example</live-example> of a JiT-compiled app, translated into French.
+
+  
+a#angular-i18n
+.l-main-section
+:marked
+  ## Angular and _i18n_ template translation
+
+  Application internationalization is a challenging, many-faceted effort that 
+  takes dedication and enduring commitment.
+  Angular's _i18n_ internationalization facilities can help.
+
+  This page describes the _i18n_ tools to assist translation of component template text
+  into multiple languages.
+
+.l-sub-section
+  :marked
+    Practitioners of _internationalization_ refer to a translatable text as a "_message_".
+    This page uses the words "_text_" and "_message_" interchangably and in the combination, "_text message_".
+  
+:marked
+  The _i18n_ template translation process has four phases:
+
+  1. Mark static text messages in your component templates for translation.
+  
+  1. An angular _i18n_ tool extracts the marked messages into an industry standard translation source file.
+
+  1. A translator edits that file, translating the extracted text messages into the target language, 
+  and returns the file to you.
+
+  1. The Angular compiler imports the completed translation files, 
+  replaces the original messages with translated text, and generates a new version of the application
+  in the target language.
+
+  You build and deploy a separate version of the application for each supported language.
+
+a#i18n-attribute
+.l-main-section
+:marked
+  ## Mark text with the _i18n_ attribute
+  
+  The Angular `i18n` attribute is a marker for translatable content.
+  Place it on every element tag whose fixed text should be translated.
+
+.alert.is-helpful
+  :marked
+    `i18n` is not an Angular _directive_. It's a custom _attribute_, recognized by Angular tools and compilers.
```

Comment:
it gets remove at compile time
_(this is actually only true when a i18n file is provided to ngc for now but eventually it will always)_